src/tmx/TmxUtils/src/RSU_MIB_4_1.h

```diff
@@ -0,0 +1,69 @@
+#pragma once
+namespace tmx
```

Comment:
You can do something like this to make this less verbose
```
namespace tmx::utils::rsu41::mid::oid
```

Reply:
Addressed https://github.com/usdot-fhwa-OPS/V2X-Hub/pull/564/commits/d46893578a0d09dfdf8decc8166c102da3e3a449